### ISSUE TITLE
[Bug Report][2.2.11] Screen reader doesn't read v-select as combo box 

### ISSUE THREAD

**Issue by Ben-SMS** [vuetifyjs/vuetify#15541]
### Environment
**Vuetify Version:** 2.2.11
**Vue Version:** 2.6.11
**Browsers:** Google Chrome
**OS:** Windows 10

### Steps to reproduce
1. Open the screen reader NVDA
2. Tab to focus the dropdown
3. Listen to the screen reader

### Expected Behavior
Should read as "Combo box" like HTML "select" element. Here is an example link https://www.w3schools.com/tags/tryit.asp?filename=tryhtml_select

### Actual Behavior
NVDA reads "... Menu button collapsed, sub-menu..."

### Reproduction Link
<a href="https://codepen.io/doanthaiphong/pen/rNdLGgP?editors=1010" target="_blank">https://codepen.io/doanthaiphong/pen/rNdLGgP?editors=1010</a>

**Comment by rscsam** [comment]
Dropped a PR @KaelWD @Ben-SMS  (or whoever). This is my first time contributing to this repo so let me know if you think I need more changes.

**Comment by Martin-Idel** [comment]
This seems to be a duplicate of https://github.com/vuetifyjs/vuetify/issues/14904 but it would be valuable to fix this!
